flink-connectors/flink-connector-kafka-0.11/src/test/java/org/apache/flink/streaming/connectors/kafka/internal/TransactionalIdsGeneratorTest.java

```diff
@@ -45,6 +45,24 @@ public void testGenerateIdsToUse() {
 			generator.generateIdsToUse(36));
 	}
 
+	@Test
+	public void testGenerateIdsWithPercentageToUse() {
+		TransactionalIdsGenerator generator = new TransactionalIdsGenerator("test%", 2, SUBTASKS_COUNT, POOL_SIZE, SAFE_SCALE_DOWN_FACTOR);
```

Comment:
This line is too long, we can fold it and make it two lines to let it more readable.